lib/src/widgets/typing/ds_dot_animation.widget.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:flutter/material.dart';
+
+import 'ds_dot.widget.dart';
+
+/// To define the stitch appearance use [size] and [color]
+/// Use [numberOfDots] to set the amount of dots and use [innerPadding] to set the spacing between them
+/// To change the animation speed use [animationTime]
+class DsDotAnimationWidget extends StatefulWidget {
+  final double size;
+  final Color color;
+  final int numberOfDots;
+  final double innerPadding;
+  final Duration animationTime;
+
+  const DsDotAnimationWidget({
+    Key? key,
+    this.numberOfDots = 4,
+    this.size = 10,
+    this.innerPadding = 2.5,
+    this.animationTime = const Duration(milliseconds: 200),
+    this.color = Colors.black,
+  }) : super(key: key);
+
+  @override
+  State<DsDotAnimationWidget> createState() => _DsDotAnimationWidget();
```

Comment:
Use the prefix 'State' to name of state class

Reply:
Done